### ISSUE TITLE
Is it possible to instantly open panel with outhput to see tests in progress?

### ISSUE THREAD

**Issue by kiddick** [kaste/PyTest#8]
Waiting for tests in background and looking just at status bar is kinda confusing.

**Comment by kaste** [comment]
That is currently not implemented. What I usually do is that I have a keyboard shortcut set

```
{ "keys": ["ctrl+'"], "command": "pytest_toggle_panel" }
```

which will show/hide the output panel. Usually, I think that automatic popping or starring at test runs as the dots fly in to be very distracting. Don't you lose the focus on writing code or thinking about what you just wrote?

**Comment by kiddick** [comment]
When I am running tests I am focused on test. So it is a bit confusing when nothing happens except small progress on status bar. So could be useful for users to have such configurable option

**Comment by kaste** [comment]
Yeah, sure, if I don't wear my TDD hat, I actually watch these dots as well, that's why I implemented this toggle panel command I just mentioned. I don't remember right now, but don't you need this command anyway to _close_ the panel and return to editing.
